duckduckgo/macos-browser
# Show badged NetP icon to users who are eligible for the waitlist

Task/Issue URL: https://app.asana.com/0/0/1205856721915108/f
Tech Design URL:
CC:

**Description**:

This PR updates the app to show the badged NetP icon to users who are eligible to join the waitlist. After they join, the icon will go away, and show back up again after they get let in.

**Steps to test this PR**:
1. Reset NetP state entirely
2. Make sure you reset the NetP feature flags to either use the remote values or be set to `ON`
3. Run the app, you should see a badged NetP icon in the toolbar
4. Click it to open the waitlist and sign up
5. After you do this, check that the badge goes away

After that you're good, the rest of the flow is the same as always.



---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
